CHANGES.rst

```diff
@@ -204,6 +204,9 @@ API Changes
   - Removed deprecated ``Pix2Sky_AZP.check_mu`` and ``Pix2Sky_SZP.check_mu``
     methods. [#6170]
 
+  - Deprecated ``GaussianAbsorption1D`` model, as it can be better represented
+    by subtracting ``Gaussian1D`` from a continuum model. [#6200]
```

Comment:
Maybe you can say a ``Const1D`` model instead of continuum, and say that this allows better control over the continuum level?